pkg/collector/py/utils.go

```diff
@@ -51,6 +51,16 @@ const (
 	pyIntegrationListFunc = "get_datadog_wheels"
 )
 
+var (
+	// implements a string set of non-intergrations with an empty stuct map
+	nonIntegrationsWheelSet = map[string]struct{}{
+		"checks_base":        {},
```

Comment:
Should this be here?

Reply:
Yeah it was previously the only blacklisted one actually